blazarclient/v1/hosts.py

```diff
@@ -72,3 +72,40 @@ def reallocate(self, host_id, values):
         resp, body = self.request_manager.put(
             '/os-hosts/%s/allocation' % host_id, body=values)
         return body['allocation']
+
+    def list_capabilities(self, detail=False, sort_by=None):
+        url = '/os-hosts/properties'
+
+        if detail:
+            url += '?detail=True'
+
+        resp, body = self.request_manager.get(url)
+        resource_properties = body['resource_properties']
+
+        # Values is a reserved word in cliff so need to rename values column.
+        if detail:
+            for p in resource_properties:
+                p['capability_values'] = p['values']
+
+        if sort_by:
+            resource_properties = sorted(resource_properties,
+                                         key=lambda l: l[sort_by])
```

Comment:
I think `operator.itemgetter` does this for you:

```python
from operator import itemgetter

sorted(resource_properties, key=itemgetter(sort_by))
```

Reply:
the client uses lambda elsewhere so just being consistent for now.